sentry-ruby/lib/sentry/utils/real_ip.rb

```diff
@@ -63,7 +65,7 @@ def ips_from(header)
       end
 
       def filter_local_addresses(ips)
```

Comment:
this method should be renamed too, as it doesn't only filter local addresses now.